theforeman.org/pipelines/release/source/smart-proxy.groovy

```diff
@@ -58,37 +74,17 @@ pipeline {
                     sourcefile_paths = generate_sourcefiles(project_name: project_name, source_type: source_type)
                 }
             }
+            post {
+                always {
+                    deleteDir()
+                }
+            }
         }
     }
 
     post {
-        success {
-            build(
-                job: "${project_name}-${git_ref}-package-release",
-                propagate: false,
-                wait: false
-            )
-        }
-
         failure {
             notifyDiscourse(env, "${project_name} source release pipeline failed:", currentBuild.description)
         }
-
-        cleanup {
-            deleteDir()
-        }
-    }
-}
-
-def run_test(args) {
-    def ruby = args.ruby
-
-    try {
-        sh "cp config/settings.yml.example config/settings.yml"
```

Comment:
isn't this step now missing?

Reply:
https://github.com/theforeman/smart-proxy/blob/develop/config/settings.yml.example only contains commented lines so it shouldn't make a difference. `theforeman.org/pipelines/test/smart-proxy.groovy` also doesn't do it so I'm assuming the unit test suite doesn't need it.

Perhaps we should enhance the smart proxy to deal with running without a config where currently it can't?

Reply:
Wfm